### ISSUE TITLE
Crop Avatar Example with c# mvc

### ISSUE THREAD

**Issue by poojan23** [fengyuanchen/cropper#578]
Hello,

Sorry for my bad English.

I loved the plugin and the controls given. I am currently using this plugin but would like to create a same style modal which is given in crop-avatar example and upload it to server using mvc. I am finding it difficult to link the "done" button with my code and returns with "error" message only.

I would like to use the modal and some of the buttons (like move, crop, clear, rotate, flip, flop etc.) to give user more control. Can someone please guide me on it?

**Comment by fengyuanchen** [comment]
Why not ask for help on StackOverflow.com?

**Comment by poojan23** [comment]
Already asked but didn't get responses yet.

http://stackoverflow.com/questions/34634862/fengyuanchen-crop-avatar-example-using-mvc

**Comment by fengyuanchen** [comment]
I am afraid that you might have to implement it by yourself by referring the [Crop Avatar](https://github.com/fengyuanchen/cropper/tree/master/examples/crop-avatar) (PHP) example.

**Comment by marmagni** [comment]
Hello fengyuanchen, 

I'm doing exactly that. 
Its possible send to server the image already cropped by client-side? 
Do I need implement on server side based on parameters, like width, height, ratio, etc?

Thank you.

**Comment by fengyuanchen** [comment]
About upload cropped image, see #489.

**Comment by poojan23** [comment]
We have solved it and got it working. Also implemented the buttons like move, crop, scale etc. Now, I am facing 2 problems.

1. In Chrome & MS Edge, when I click the buttons respond time is very slow and have to do multiple clicks to make it work once. Is there is a solution to it?

2. I have assigned keyboard keys to the buttons I am using but for scale, it only scales to -1 in both X and Y direction. The code I am using is "this.$img.cropper('scaleX', -1, 1);" this.$img.cropper('scaleY', -1, 1);

Clicking the icon issue is solves with the issue #500 but can't work with keydown.
